README.md

```diff
@@ -104,7 +104,7 @@ is to feel natural for somebody used to coding in Python, and reasonably
 easy to learn for somebody who is not a pythonista.
 
 The dependencies of the operator framework are kept as minimal as possible;
-currently that's Python 3.5 or greater, and `PyYAML` (both are included by
+currently that's Python 3.6 or greater, and `PyYAML` (both are included by
 default in Ubuntu's cloud images from 16.04 on).
```

Comment:
This should say 18.04 (bionic) now. The default there is Python 3.6, by the looks: https://packages.ubuntu.com/bionic/python3

I haven't confirmed that PyYAML is included in Ubuntu's cloud image for 18.04, though.

Reply:
Nice catch. I'll update the version and verify against the cloud image.